brianlong/www.validators.app
# 187087318 - skip network if stake pools don't exist

#### What's this PR do?
- As in the title.

#### How should this be manually tested?
- run `rails script/gather_stake_accounts.rb` and check if the pipeline is run only for the network where stake pools exist.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187087318)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
